.github/workflows/npx-server-smoke.yml

```
@@ -60,7 +60,12 @@ jobs:
     runs-on: ${{ matrix.runner }}
     timeout-minutes: 35
     env:
-      LANGWATCH_HOME: ${{ github.workspace }}/.langwatch-smoke
+      # Outside $GITHUB_WORKSPACE on purpose (see the "Run npx @langwatch/server
+      # in a sandbox" step below): Node walks every ancestor directory looking
+      # for a node_modules, and the checkout's own workspace install would sit
+      # on that ancestor chain if LANGWATCH_HOME or the extracted tarball lived
+      # under the workspace.
+      LANGWATCH_HOME: ${{ runner.temp }}/.langwatch-smoke
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)